README.md

```diff
@@ -787,6 +787,77 @@ func main() {
 }
 ```
 
+### Roadrunner middleware
+To use Souin as roadrunner middleware, you have to build your `rr` binary with the souin dependency.
+```toml
+[velox]
+build_args = ['-trimpath', '-ldflags', '-s -X github.com/roadrunner-server/roadrunner/v2/internal/meta.version=v2.10.7 -X github.com/roadrunner-server/roadrunner/v2/internal/meta.buildTime=10:00:00']
```

Comment:
You may use env variables to configure version, buildTime, and tokens, like:
`build_args = ['-trimpath', '-ldflags', '-s -X github.com/roadrunner-server/roadrunner/v2/internal/meta.version=${VERSION} -X github.com/roadrunner-server/roadrunner/v2/internal/meta.buildTime=${TIME}']`

And then use:

```bash
VERSION=v2.10.7 TIME=(date +%H:%M) vx build -c velox.toml -o "."
```